File: SomeSums.c
#include<stdio.h>
int main(void){
  int N, A, B, sum, ans = 0;
  scanf("%d %d %d", &N, &A, &B);
  for(int i=0;i<=N;i++){
    int j=i;
    sum=0;
    while(j!=0){
      sum=sum+(j%10);
      j=j/10;
    }
    if(sum>=A && sum<=B){
      ans=ans+i;
    }
  }
  printf("%d\n", ans);
  return 0;
}

File: SomeSums.py
def main():
    N, A, B = [int(_) for _ in input().split()]
    ans = 0
    for i in range(N + 1):
        j = i
        sum_ = 0
        while j != 0:
            sum_ = sum_ + (j % 10)
            j = j // 10

        if A <= sum_ <= B:
            ans = ans + i

    print(ans)

main()
